src/doc/book/mutability.md

```diff
@@ -55,6 +55,8 @@ fn foo(mut x: i32) {
 # }
 ```
 
+Note that here, the `x` is mutable, but not the `y`.
```

Comment:
The previous paragraph already has "Note that" -- maybe this one can simply be put into the code example as a comment?